x-pack/plugin/src/test/resources/rest-api-spec/test/runtime_fields/101_geo_point_from_source.yml

```diff
@@ -14,9 +14,6 @@ setup:
             properties:
               timestamp:
                 type: date
-              location:
-                type: object
-                enabled: false
```

Comment:
Is there a reason why here you haven't applied dynamic: false like in other tests?